### DISCUSSION TITLE
Is there a Freya-15 CSS Change Summary Available?

### DISCUSSION THREAD

**Discussion by aabot99** [primefaces/community#650] [PrimeNG Templates] [answered]
Hi, I have been working with freya template for a while.  In the v15 release, there was a shift in the CSS layout design.  Before all I had to do for updating was copying the new template's assets/layout and assets/sass folder into my build, now with the shift in v15 I can't do that.  I was able to upgrade to v16 of primeng, but stuck at version 16.0.2 because of a few styling issues which I wonder is because of the v15 copy of the assets/layout and assets/sass folder. 

Is there a write-up available that describes what changes I need in my code to go to the new CSS layout design outside or I just have to walk through the code?  

UPDATE: I am going through the code changes and it is a lot.  Would you have any recommendations on how to update the old CSS approach of v15 for primeng v16 and later v17?

**Comment by aabot99** [comment, marked as answer]
I went ahead and upgraded primeng to 16.9.1 so far the styling looks right with the older v15 freya css styling except for the p-inputNumber.  I saw in a different discussion about the @layer primeng addition in 16.4.0.   I looked in the debugger and the styling issue does not appear to be coming from the old v15 freya css code, but in the new @layer code.  If I insert a height on each arrow button I can fix the styling.  Will there be a v16.9 update to fix the arrow buttons.  Thanks.

![image](https://github.com/primefaces/community/assets/152433894/f3c17e25-646a-449f-a4d1-bb10e4f2d518)

**Reply by aabot99** [reply]
No longer an issue.  Created a work-around.

**Reply by ThoSap** [reply]
@aabot99 FreyaNG 17.0.0 is available since 27.12.2023, and this is no longer an issue.

**Reply by aabot99** [reply]
Unfortunately, my code was built with the old freya styling design that changed in v15.  I have not been able to switch over to the new style since it breaks a few things.  Are thing any instructions to get the old freya styling (prior to v15) working with v17?  Right now, I am I just working around the styling oddities.

**Comment by cetincakiroglu** [comment]
Hi,

We'll update templates to v17 in the near future and we'll add documentation about the CSS layer. A similar question related to template update is answered here, maybe it would help.
[Mirage | Upgrade v13.4.1 to v16.4.1 bugs · primefaces · Discussion \#130](https://github.com/orgs/primefaces/discussions/130%23discussioncomment-7130755)
